themes/bootprint3/less/common.less

```diff
@@ -46,6 +46,8 @@ h2 { margin: 8px 0; }
   background: #fff;
 }
 
+h2 { margin: 0 8px 8px; }
```

Comment:
Are the changes to bootprint3 / bootstrap3 less files here actually directly related to the modern theme, or are these either legacy code that should be reverted or new changes that should be separately merged to master?